FluidNC/src/Machine/MachineConfig.cpp

```diff
@@ -79,6 +79,7 @@ namespace Machine {
 
         if (_userOutputs == nullptr) {
             _userOutputs = new UserOutputs();
+            _userOutputs->init();
```

Comment:
I think this is unnecessary since I move the array population to the UserOutputs constructor.

Reply:
OK.